components/Players/PlayerManager.vue

```diff
@@ -341,8 +343,11 @@ export default Vue.extend({
       'setPreviousTrack',
       'setLastItemIndex',
       'playPause',
+      'pause',
+      'play',
```

Comment:
```suggestion
      'unpause',
```

See my other comment